### DISCUSSION TITLE
How much is the RAM and Flash usage ?

### DISCUSSION THREAD

**Discussion by jaytxrx** [OpenI6X/opentx#325] [General]
This is just for information purpose.

I am curious how much is the current Ram and Flash usage. Can anyone share this info ? A rough value is also fine.

Will i6x be able to handle future opentx changes ? If Ram and Flash are almost used, it won't be possible to integrate future changes - right ?

**Comment by megacct** [comment]
From the latest std-backlight build:

Flash used: 125500 / 131072 (95%), left: 5572
RAM used: 15916 / 16384 (97%), left: 468

I can't speak to future changes

**Comment by ajjjjjjjj** [comment]
Exactly as @megacct  written, using new compiler with LTO there is considerable amount of space for future changes.

**Comment by jaytxrx** [comment]
I see here that there are some savings in ram and flash due to new compiler. Can you please provide an update regarding the current usage of ram and flash ? @ajjjjjjjj 

https://github.com/OpenI6X/opentx/discussions/326

**Reply by ajjjjjjjj** [reply]
See @megacct post. Those are with new compiler.

**Reply by jaytxrx** [reply]
Oh.. I thought those were the values without using the new compiler
